recipes/skaware/skalibs.inc

```diff
@@ -0,0 +1,66 @@
+DESCRIPTION = "skalibs is a package centralizing the free software / open source C development files used for building all software at skarnet.org"
+LICENSE = "ISC"
+
+SRC_URI = "http://skarnet.org/software/${PN}/${PN}-${PV}.tar.gz"
+
+inherit c make
+
+endianness = "${@['big', 'little'][d.get('HOST_ENDIAN') == 'l']}"
+
+SRC_URI += "file://sysdeps file://sysdeps.h"
+do_configure[prefuncs] += "do_configure_sysdeps"
+do_configure_sysdeps() {
```

Comment:
please add some documentation about all this, to give the next poor guy a chance :)